src/main/java/teammates/common/util/ActivityLogEntry.java

```diff
@@ -139,7 +140,7 @@ public ActivityLogEntry(AccountAttributes userAccount, boolean isMasquerade, Str
         }
         action = servletName; //TODO: remove this?
         toShow = true;
-        message = logMessage;
+        message = logMessage.isEmpty() ? "null" : StringHelper.join("", logMessage.toArray(new String[logMessage.size()]));
```

Comment:
Doesn't this change its behaviour? The original function doesn't do anything if its empty.